osu.Game/Overlays/Rankings/Displays/CountriesDisplay.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Game.Online.API;
+using osu.Game.Online.API.Requests;
+using osu.Game.Overlays.Rankings.Tables;
+
+namespace osu.Game.Overlays.Rankings.Displays
+{
+    public class CountriesDisplay : RankingsDisplay
```

Comment:
I was very confused as to what this is. needs to be called `CountryRankingsDisplay`